xrpl/transaction/tx.go

```diff
@@ -115,7 +115,7 @@ func (tx *BaseTx) Flatten() FlatTransaction {
 		flattened["Fee"] = tx.Fee.String()
 	}
 	if tx.Sequence != 0 {
-		flattened["Sequence"] = tx.Sequence
+		flattened["Sequence"] = int(tx.Sequence)
```

Comment:
Why do we cast it to an `int` when it's an `uint32` in the struct ?
I have the same questions for the other fields below.